Source/SourceKittenFramework/Request.swift

```diff
@@ -80,27 +80,39 @@ private func fromSourceKit(sourcekitObject: sourcekitd_variant_t) -> SourceKitRe
     case SOURCEKITD_VARIANT_TYPE_STRING:
         let length = sourcekitd_variant_string_get_length(sourcekitObject)
         let ptr = sourcekitd_variant_string_get_ptr(sourcekitObject)
-        let string = swiftStringFrom(ptr, length: length)
+        let string = swiftStringFrom(bytes: ptr!, length: length)
         return string
     case SOURCEKITD_VARIANT_TYPE_INT64:
         return sourcekitd_variant_int64_get_value(sourcekitObject)
     case SOURCEKITD_VARIANT_TYPE_BOOL:
         return sourcekitd_variant_bool_get_value(sourcekitObject)
     case SOURCEKITD_VARIANT_TYPE_UID:
-        return stringForSourceKitUID(sourcekitd_variant_uid_get_value(sourcekitObject))!
+        return stringForSourceKitUID(sourcekitd_variant_uid_get_value(sourcekitObject)!)!
```

Comment:
`sourcekitd_` functions returns result with `@_autounwrapped` attriute.
We don't need `!` on using returned value directly.
So, we can reduce diffs from master with following:
```diff
diff --git a/Source/SourceKittenFramework/Request.swift b/Source/SourceKittenFramework/Request.swift
index f3dc40f..c88cec3 100644
--- a/Source/SourceKittenFramework/Request.swift
+++ b/Source/SourceKittenFramework/Request.swift
@@ -87,7 +87,7 @@ private func fromSourceKit(_ sourcekitObject: sourcekitd_variant_t) -> SourceKit
     case SOURCEKITD_VARIANT_TYPE_BOOL:
         return sourcekitd_variant_bool_get_value(sourcekitObject)
     case SOURCEKITD_VARIANT_TYPE_UID:
-        return stringForSourceKitUID(sourcekitd_variant_uid_get_value(sourcekitObject)!)!
+        return stringForSourceKitUID(sourcekitd_variant_uid_get_value(sourcekitObject))!
     case SOURCEKITD_VARIANT_TYPE_NULL:
         return nil
     default:
@@ -214,86 +214,86 @@ public enum Request {
         case .EditorOpen(let file):
             if let path = file.path {
                 dict = [
-                    sourcekitd_uid_get_from_cstr("key.request")!: sourcekitd_request_uid_create(sourcekitd_uid_get_from_cstr("source.request.editor.open")!)!,
-                    sourcekitd_uid_get_from_cstr("key.name")!: sourcekitd_request_string_create(path)!,
-                    sourcekitd_uid_get_from_cstr("key.sourcefile")!: sourcekitd_request_string_create(path)!
+                    sourcekitd_uid_get_from_cstr("key.request"): sourcekitd_request_uid_create(sourcekitd_uid_get_from_cstr("source.request.editor.open")),
+                    sourcekitd_uid_get_from_cstr("key.name"): sourcekitd_request_string_create(path),
+                    sourcekitd_uid_get_from_cstr("key.sourcefile"): sourcekitd_request_string_create(path)
                 ]
             } else {
                 dict = [
-                    sourcekitd_uid_get_from_cstr("key.request")!: sourcekitd_request_uid_create(sourcekitd_uid_get_from_cstr("source.request.editor.open")!)!,
-                    sourcekitd_uid_get_from_cstr("key.name")!: sourcekitd_request_string_create(String(file.contents.hash))!,
-                    sourcekitd_uid_get_from_cstr("key.sourcetext")!: sourcekitd_request_string_create(file.contents)!
+                    sourcekitd_uid_get_from_cstr("key.request"): sourcekitd_request_uid_create(sourcekitd_uid_get_from_cstr("source.request.editor.open")),
+                    sourcekitd_uid_get_from_cstr("key.name"): sourcekitd_request_string_create(String(file.contents.hash)),
+                    sourcekitd_uid_get_from_cstr("key.sourcetext"): sourcekitd_request_string_create(file.contents)
                 ]
             }
         case .CursorInfo(let file, let offset, let arguments):
             var compilerargs = arguments.map({ sourcekitd_request_string_create($0) })
             dict = [
-                sourcekitd_uid_get_from_cstr("key.request")!: sourcekitd_request_uid_create(sourcekitd_uid_get_from_cstr("source.request.cursorinfo")!)!,
-                sourcekitd_uid_get_from_cstr("key.name")!: sourcekitd_request_string_create(file)!,
-                sourcekitd_uid_get_from_cstr("key.sourcefile")!: sourcekitd_request_string_create(file)!,
-                sourcekitd_uid_get_from_cstr("key.offset")!: sourcekitd_request_int64_create(offset)!,
-                sourcekitd_uid_get_from_cstr("key.compilerargs")!: sourcekitd_request_array_create(&compilerargs, compilerargs.count)!
+                sourcekitd_uid_get_from_cstr("key.request"): sourcekitd_request_uid_create(sourcekitd_uid_get_from_cstr("source.request.cursorinfo")),
+                sourcekitd_uid_get_from_cstr("key.name"): sourcekitd_request_string_create(file),
+                sourcekitd_uid_get_from_cstr("key.sourcefile"): sourcekitd_request_string_create(file),
+                sourcekitd_uid_get_from_cstr("key.offset"): sourcekitd_request_int64_create(offset),
+                sourcekitd_uid_get_from_cstr("key.compilerargs"): sourcekitd_request_array_create(&compilerargs, compilerargs.count)
             ]
         case .CustomRequest(let request):
             return request
         case .CodeCompletionRequest(let file, let contents, let offset, let arguments):
             var compilerargs = arguments.map({ sourcekitd_request_string_create($0) })
             dict = [
-                sourcekitd_uid_get_from_cstr("key.request")!: sourcekitd_request_uid_create(sourcekitd_uid_get_from_cstr("source.request.codecomplete")!)!,
-                sourcekitd_uid_get_from_cstr("key.name")!: sourcekitd_request_string_create(file)!,
-                sourcekitd_uid_get_from_cstr("key.sourcefile")!: sourcekitd_request_string_create(file)!,
-                sourcekitd_uid_get_from_cstr("key.sourcetext")!: sourcekitd_request_string_create(contents)!,
-                sourcekitd_uid_get_from_cstr("key.offset")!: sourcekitd_request_int64_create(offset)!,
-                sourcekitd_uid_get_from_cstr("key.compilerargs")!: sourcekitd_request_array_create(&compilerargs, compilerargs.count)!
+                sourcekitd_uid_get_from_cstr("key.request"): sourcekitd_request_uid_create(sourcekitd_uid_get_from_cstr("source.request.codecomplete")),
+                sourcekitd_uid_get_from_cstr("key.name"): sourcekitd_request_string_create(file),
+                sourcekitd_uid_get_from_cstr("key.sourcefile"): sourcekitd_request_string_create(file),
+                sourcekitd_uid_get_from_cstr("key.sourcetext"): sourcekitd_request_string_create(contents),
+                sourcekitd_uid_get_from_cstr("key.offset"): sourcekitd_request_int64_create(offset),
+                sourcekitd_uid_get_from_cstr("key.compilerargs"): sourcekitd_request_array_create(&compilerargs, compilerargs.count)
             ]
         case .Interface(let file, let uuid):
             let arguments = ["-x", "objective-c", file, "-isysroot", sdkPath()]
             var compilerargs = arguments.map({ sourcekitd_request_string_create($0) })
             dict = [
-                sourcekitd_uid_get_from_cstr("key.request")!: sourcekitd_request_uid_create(sourcekitd_uid_get_from_cstr("source.request.editor.open.interface.header")!)!,
-                sourcekitd_uid_get_from_cstr("key.name")!: sourcekitd_request_string_create(uuid)!,
-                sourcekitd_uid_get_from_cstr("key.filepath")!: sourcekitd_request_string_create(file)!,
-                sourcekitd_uid_get_from_cstr("key.compilerargs")!: sourcekitd_request_array_create(&compilerargs, compilerargs.count)!
+                sourcekitd_uid_get_from_cstr("key.request"): sourcekitd_request_uid_create(sourcekitd_uid_get_from_cstr("source.request.editor.open.interface.header")),
+                sourcekitd_uid_get_from_cstr("key.name"): sourcekitd_request_string_create(uuid),
+                sourcekitd_uid_get_from_cstr("key.filepath"): sourcekitd_request_string_create(file),
+                sourcekitd_uid_get_from_cstr("key.compilerargs"): sourcekitd_request_array_create(&compilerargs, compilerargs.count)
             ]
         case .FindUSR(let file, let usr):
             dict = [
-                sourcekitd_uid_get_from_cstr("key.request")!: sourcekitd_request_uid_create(sourcekitd_uid_get_from_cstr("source.request.editor.find_usr")!)!,
-                sourcekitd_uid_get_from_cstr("key.usr")!: sourcekitd_request_string_create(usr)!,
-                sourcekitd_uid_get_from_cstr("key.sourcefile")!: sourcekitd_request_string_create(file)!
+                sourcekitd_uid_get_from_cstr("key.request"): sourcekitd_request_uid_create(sourcekitd_uid_get_from_cstr("source.request.editor.find_usr")),
+                sourcekitd_uid_get_from_cstr("key.usr"): sourcekitd_request_string_create(usr),
+                sourcekitd_uid_get_from_cstr("key.sourcefile"): sourcekitd_request_string_create(file)
             ]
         case .Index(let file, let arguments):
             var compilerargs = arguments.map({ sourcekitd_request_string_create($0) })
             dict = [
-                sourcekitd_uid_get_from_cstr("key.request")!: sourcekitd_request_uid_create(sourcekitd_uid_get_from_cstr("source.request.indexsource")!)!,
-                sourcekitd_uid_get_from_cstr("key.sourcefile")!: sourcekitd_request_string_create(file)!,
-                sourcekitd_uid_get_from_cstr("key.compilerargs")!: sourcekitd_request_array_create(&compilerargs, compilerargs.count)!
+                sourcekitd_uid_get_from_cstr("key.request"): sourcekitd_request_uid_create(sourcekitd_uid_get_from_cstr("source.request.indexsource")),
+                sourcekitd_uid_get_from_cstr("key.sourcefile"): sourcekitd_request_string_create(file),
+                sourcekitd_uid_get_from_cstr("key.compilerargs"): sourcekitd_request_array_create(&compilerargs, compilerargs.count)
             ]
         case .Format(let file, let line, let useTabs, let indentWidth):
-            let formatOptions: [sourcekitd_uid_t: sourcekitd_object_t?] = [
-                sourcekitd_uid_get_from_cstr("key.editor.format.indentwidth")!: sourcekitd_request_int64_create(indentWidth)!,
-                sourcekitd_uid_get_from_cstr("key.editor.format.tabwidth")!: sourcekitd_request_int64_create(indentWidth)!,
-                sourcekitd_uid_get_from_cstr("key.editor.format.usetabs")!: sourcekitd_request_int64_create(useTabs ? 1 : 0)!,
+            let formatOptions = [
+                sourcekitd_uid_get_from_cstr("key.editor.format.indentwidth"): sourcekitd_request_int64_create(indentWidth),
+                sourcekitd_uid_get_from_cstr("key.editor.format.tabwidth"): sourcekitd_request_int64_create(indentWidth),
+                sourcekitd_uid_get_from_cstr("key.editor.format.usetabs"): sourcekitd_request_int64_create(useTabs ? 1 : 0),
             ]
             var formatOptionsKeys = Array(formatOptions.keys.map({ $0 as sourcekitd_uid_t? }))
             var formatOptionsValues = Array(formatOptions.values)
             dict = [
-                sourcekitd_uid_get_from_cstr("key.request")!: sourcekitd_request_uid_create(sourcekitd_uid_get_from_cstr("source.request.editor.formattext")!)!,
-                sourcekitd_uid_get_from_cstr("key.name")!: sourcekitd_request_string_create(file)!,
-                sourcekitd_uid_get_from_cstr("key.line")!: sourcekitd_request_int64_create(line)!,
-                sourcekitd_uid_get_from_cstr("key.editor.format.options")!: sourcekitd_request_dictionary_create(&formatOptionsKeys, &formatOptionsValues, formatOptions.count)!
+                sourcekitd_uid_get_from_cstr("key.request"): sourcekitd_request_uid_create(sourcekitd_uid_get_from_cstr("source.request.editor.formattext")),
+                sourcekitd_uid_get_from_cstr("key.name"): sourcekitd_request_string_create(file),
+                sourcekitd_uid_get_from_cstr("key.line"): sourcekitd_request_int64_create(line),
+                sourcekitd_uid_get_from_cstr("key.editor.format.options"): sourcekitd_request_dictionary_create(&formatOptionsKeys, &formatOptionsValues, formatOptions.count)
             ]
         case .ReplaceText(let file, let offset, let length, let sourceText):
             dict = [
-                sourcekitd_uid_get_from_cstr("key.request")!: sourcekitd_request_uid_create(sourcekitd_uid_get_from_cstr("source.request.editor.replacetext")!)!,
-                sourcekitd_uid_get_from_cstr("key.name")!: sourcekitd_request_string_create(file)!,
-                sourcekitd_uid_get_from_cstr("key.offset")!: sourcekitd_request_int64_create(offset)!,
-                sourcekitd_uid_get_from_cstr("key.length")!: sourcekitd_request_int64_create(length)!,
-                sourcekitd_uid_get_from_cstr("key.sourcetext")!: sourcekitd_request_string_create(sourceText)!,
+                sourcekitd_uid_get_from_cstr("key.request"): sourcekitd_request_uid_create(sourcekitd_uid_get_from_cstr("source.request.editor.replacetext")),
+                sourcekitd_uid_get_from_cstr("key.name"): sourcekitd_request_string_create(file),
+                sourcekitd_uid_get_from_cstr("key.offset"): sourcekitd_request_int64_create(offset),
+                sourcekitd_uid_get_from_cstr("key.length"): sourcekitd_request_int64_create(length),
+                sourcekitd_uid_get_from_cstr("key.sourcetext"): sourcekitd_request_string_create(sourceText),
             ]
         }
         var keys = Array(dict.keys.map({ $0 as sourcekitd_uid_t? }))
         var values = Array(dict.values)
-        return sourcekitd_request_dictionary_create(&keys, &values, dict.count)!
+        return sourcekitd_request_dictionary_create(&keys, &values, dict.count)
     }
 
     /**
@@ -334,9 +334,9 @@ public enum Request {
     */
     public func send() -> [String: SourceKitRepresentable] {
         initializeSourceKit
-        let response = sourcekitd_send_request_sync(sourcekitObject)
-        defer { sourcekitd_response_dispose(response!) }
-        return fromSourceKit(sourcekitd_response_get_value(response!)) as! [String: SourceKitRepresentable]
+        let response = sourcekitd_send_request_sync(sourcekitObject)!
+        defer { sourcekitd_response_dispose(response) }
+        return fromSourceKit(sourcekitd_response_get_value(response)) as! [String: SourceKitRepresentable]
     }
 
     /// A enum representation of SOURCEKITD_ERROR_*
@@ -363,7 +363,7 @@ public enum Request {
         }
         
         fileprivate init(response: sourcekitd_response_t) {
-            let description = String(validatingUTF8: sourcekitd_response_error_get_description(response)!)
+            let description = String(validatingUTF8: sourcekitd_response_error_get_description(response))
             switch sourcekitd_response_error_get_kind(response) {
             case SOURCEKITD_ERROR_CONNECTION_INTERRUPTED: self = .ConnectionInterrupted(description)
             case SOURCEKITD_ERROR_REQUEST_INVALID: self = .Invalid(description)
@@ -382,16 +382,16 @@ public enum Request {
     */
     public func failableSend() throws -> [String: SourceKitRepresentable] {
         initializeSourceKitFailable
-        let response = sourcekitd_send_request_sync(sourcekitObject)
-        defer { sourcekitd_response_dispose(response!) }
-        if sourcekitd_response_is_error(response!) {
-            let error = Request.Error(response: response!)
+        let response = sourcekitd_send_request_sync(sourcekitObject)!
+        defer { sourcekitd_response_dispose(response) }
+        if sourcekitd_response_is_error(response) {
+            let error = Request.Error(response: response)
             if case .ConnectionInterrupted = error {
                 _ = sourceKitWaitingRestoredSemaphore.wait(timeout: DispatchTime.now() + 10)
             }
             throw error
         }
-        return fromSourceKit(sourcekitd_response_get_value(response!)) as! [String: SourceKitRepresentable]
+        return fromSourceKit(sourcekitd_response_get_value(response)) as! [String: SourceKitRepresentable]
     }
 }
 
@@ -399,20 +399,20 @@ public enum Request {
 
 extension Request: CustomStringConvertible {
     /// A textual representation of `Request`.
-    public var description: String { return String(validatingUTF8: sourcekitd_request_description_copy(sourcekitObject)!)! }
+    public var description: String { return String(validatingUTF8: sourcekitd_request_description_copy(sourcekitObject))! }
 }
 
 private func interfaceForModule(module: String, compilerArguments: [String]) -> [String: SourceKitRepresentable] {
     var compilerargs = compilerArguments.map { sourcekitd_request_string_create($0) }
     let dict: [sourcekitd_uid_t: sourcekitd_object_t?] = [
-        sourcekitd_uid_get_from_cstr("key.request")!: sourcekitd_request_uid_create(sourcekitd_uid_get_from_cstr("source.request.editor.open.interface")!)!,
-        sourcekitd_uid_get_from_cstr("key.name")!: sourcekitd_request_string_create(NSUUID().uuidString)!,
-        sourcekitd_uid_get_from_cstr("key.compilerargs")!: sourcekitd_request_array_create(&compilerargs, compilerargs.count)!,
-        sourcekitd_uid_get_from_cstr("key.modulename")!: sourcekitd_request_string_create("SourceKittenFramework.\(module)")!
+        sourcekitd_uid_get_from_cstr("key.request"): sourcekitd_request_uid_create(sourcekitd_uid_get_from_cstr("source.request.editor.open.interface")),
+        sourcekitd_uid_get_from_cstr("key.name"): sourcekitd_request_string_create(NSUUID().uuidString),
+        sourcekitd_uid_get_from_cstr("key.compilerargs"): sourcekitd_request_array_create(&compilerargs, compilerargs.count),
+        sourcekitd_uid_get_from_cstr("key.modulename"): sourcekitd_request_string_create("SourceKittenFramework.\(module)")
     ]
     var keys = Array(dict.keys.map({ $0 as sourcekitd_uid_t? }))
     var values = Array(dict.values)
-    return Request.CustomRequest(sourcekitd_request_dictionary_create(&keys, &values, dict.count)!).send()
+    return Request.CustomRequest(sourcekitd_request_dictionary_create(&keys, &values, dict.count)).send()
 }
 
 internal func libraryWrapperForModule(module: String, loadPath: String, spmModule: String, compilerArguments: [String]) -> String {
```